competencySyncScript

```diff
@@ -45,6 +53,116 @@ function testRequests() {
   }
 }
 
+function getChangedDriveFiles(folders) {
+  var folders = ["testFolder"];
+  var folder = DriveApp.getFoldersByName(folders[0]);
+  
+  while (folder.hasNext()) {
+    var files = folder.next().getFiles();
+    while (files.hasNext()) { //for each file in the folder
+      var file = files.next();
+      var fileData = {
+        "fileName" : file.getName()
+        //"fileDecodedContent" : file.getAs(contentType) //HOW DO I GET AS A STRING???????
+      };
+      
+      var fileBlob = file.getBlob();
+      var output = fileBlob.getDataAsString();
+      
+      var fileText = file.getAs(MimeType.PDF);
+      Logger.log(fileText);
+    }
+  }
+}
+
+function createCommit(fileName, fileContent) {
+  fileName = "testFile1";
+  fileContent = "HELOELOELOELOLEO";
+  //Create data:
+  var commitMessage = "Updated " + fileName + ".";
+  var branchName = "branch-" + prCounter.toString();
+  var fileContent = encodeBase64Str(fileContent);
+  var url = "https://api.github.com/repos/sendwithus/" + repoName + "/contents/" + fileName +"?ref=master&access_token=" + personalAccessToken;
+  var fileData = UrlFetchApp.fetch(url);
+  fileData = JSON.parse(fileData);
+  var sha = fileData.sha;
+  
+  if (branchExists(branchName) == false) {
+    //create new branch with name branchName
+  }
+  
+  var content = {
+    "message" : commitMessage,
+    "content" : fileContent,
+    "sha" : sha,
+    "branch" : branchName
+  };
+  
+  var options = {
```

Comment:
You can probably extract this object out into a function called `createOptions` that you pass in the content to.

Reply:
Done.